cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/ProgramLifecycleHttpHandler.java

```diff
@@ -1056,9 +1017,7 @@ public void deleteFlowQueues(HttpRequest request, HttpResponder responder,
     Id.Program programId = Id.Program.from(namespaceId, appId, ProgramType.FLOW, flowId);
     try {
       ProgramStatus status = getProgramStatus(programId);
-      if (status.getStatus().equals(HttpResponseStatus.NOT_FOUND.toString())) {
```

Comment:
This is redundant condition, since the `getProgramStatus` throws `NotFoundException`.